src/components/panels/EventsPanel.vue

```diff
@@ -64,16 +82,24 @@
 import NoDataItem from '@/components/NoDataItem.vue';
 import HubPanelWrap from '@/components/HubPanelWrap';
 import Collapsible from '@/components/Collapsible';
+import TimeBadge from '@/components/TimeBadge.vue';
 
 import util from '@/utilities';
 
+const reversedHealthEvents = ['Thermia Fractures'];
+
 export default {
   name: 'EventsPanel',
   props: ['events'],
   computed: {
     headertext() {
       return 'Events';
     },
+    activeEvents() {
+      return this.$props.events.filter((event) => {
```

Comment:
should just be a single line
```suggestion
      return this.$props.events.filter((event) => event.active);
```